src/z3c/jbot/manager.py

```diff
@@ -131,6 +135,7 @@ def registerTemplate(self, template, token):
         filename = path.replace(os.path.sep, '.')
         if filename not in paths:
             self.templates[token] = IGNORE
+            template._v_last_read = False
```

Comment:
Whoa, what's this?  I don't remember this from earlier reviews (but maybe I just overlooked it).

I don't understand why this is necessary for Python 3 only?  Or was it a bug that was also present on Python 2?

What I remember about _v_ attributes is that code should expect them to disappear at any random time, so is this for replacing a possible stale ` _v_last_read == True` value with a `False` to force a template reload or something?

Does this fix need a regression test?  A mention in a changelog?

Reply:
@pbauer can you give us an update here?

Reply:
Commit a72705e66e7a925506f35436cbbc3284934a2313 title is "Be sure ignored templates will reread from disk".
IIRC removing this line was causing a test to fail on Py3 (not sure in Py2 but I think not).

Reply:
This is the failing test that breaks if I revert that commit:
```
[ale@emily z3c.jbot]$ git revert a72705e66e7a925506f35436cbbc3284934a2313
[python3 5e156e3] Revert "Be sure ignored templates will reread from disk"
 1 file changed, 1 insertion(+), 5 deletions(-)
[ale@emily z3c.jbot]$ ./bin/test 
...

Failure in test /home/ale/Code/plone/packages/z3c.jbot/src/z3c/jbot/README.txt
Failed doctest test for README.txt
  File "/home/ale/Code/plone/packages/z3c.jbot/src/z3c/jbot/README.txt", line 0

----------------------------------------------------------------------
File "/home/ale/Code/plone/packages/z3c.jbot/src/z3c/jbot/README.txt", line 140, in README.txt
Failed example:
    view.template()
Expected:
    u'This is an example page template.\n'
Got:
    'Override from ./request.\n'

  Ran 1 tests with 1 failures, 0 errors and 0 skipped in 0.045 seconds.
Running Testing.ZopeTestCase.layer.ZopeLite tests:
  Tear down zope.testrunner.layer.UnitTests in 0.000 seconds.
  Set up Testing.ZopeTestCase.layer.ZopeLite in 0.027 seconds.
  Ran 2 tests with 1 failures, 0 errors and 0 skipped in 0.078 seconds.
Tearing down left over layers:
  Tear down Testing.ZopeTestCase.layer.ZopeLite in 0.000 seconds.
Total: 3 tests, 1 failures, 0 errors and 0 skipped in 0.224 seconds.
```

Reply:
_v_last_read is something that is around since 16 years... https://github.com/zopefoundation/zope.pagetemplate/commit/38489224613849c4caa57b1da915e9f3b42937ea Apparently it was not checked properly before and the template was always reread.

Reply:
I added a line to the changelog